starter/slack-spring-boot/src/main/kotlin/com/kreait/slack/broker/store/user/UserManager.kt

```diff
@@ -0,0 +1,62 @@
+package com.kreait.slack.broker.store.user
+
+import com.kreait.slack.api.SlackClient
+import com.kreait.slack.api.contract.jackson.common.types.Member
+import com.kreait.slack.api.contract.jackson.group.users.ListAllRequest
+import com.kreait.slack.broker.store.team.Team
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+
+class UserManager @Autowired constructor(private val slackClient: SlackClient,
```

Comment:
Is it maybe better to autoregister a simple Installation receiver that does this? instead of writing a new type of strong dependency into the installation broker?